compiler/src/openxla/compiler/nvgpu/Dialect/CUDNN/IR/CUDNNOps.td

```diff
@@ -191,4 +195,101 @@ def CUDNN_GetCurrentHandle : CUDNN_Op<"get_current_handle", [Pure]> {
     let assemblyFormat = [{ attr-dict type($handle) }];
 }
 
+//===----------------------------------------------------------------------===//
+// cudnn.graph operation
+//===----------------------------------------------------------------------===//
+
+def CUDNN_GraphOp : CUDNN_Op<"graph", [
```

Comment:
Would it make sense for this op to have `SingleBlockImplicitTerminator<"openxla::compiler::nvgpu::cudnn::ReturnOp">` as well?